ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/IRToCK3_aboriginal.txt

```diff
@@ -25,10 +26,12 @@ nuragic = {
 	color = hsv { 0.2 0.25 0.5 }
 
 	heritage = heritage_aboriginal
-	# language = language_nuragic
+	language = language_nuragic
+	ethos = ethos_egalitarian # No idea

```

Comment:
Could you create an issue with a list of cultures that have placeholder ethoses? I can do some research for them for release 3.1.

Reply:
Done